README.md

```diff
@@ -4,8 +4,7 @@
 
 **The Raycast Clone Your Linux Desktop Deserves**
 
-> **"The failed uncle of the family"**  
-> _Because every family has that one uncle... you know, the one who's a bit of a screw-up but still kind of lovable in his own tragic way. That's Backslash, and Raycast is the golden child. But deep down, we all have a soft spot for the underdog. Right?_
+> **"The failed uncle of the family"** > _Because every family has that one uncle... you know, the one who's a bit of a screw-up but still kind of lovable in his own tragic way. That's Backslash, and Raycast is the golden child. But deep down, we all have a soft spot for the underdog. Right?_
```

Comment:
Return to the line here, at `> _Because`? :thinking:

Reply:
Prettier does this when using the format command 🐳

Reply:
A bit stupid, a solution is adding the readme to the `prettierignore`?

Reply:
Yep, good idea. I merged this here : #24 :+1:

Reply:
I have updated the branch